src/modules/store.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable quotes */
+const InnerTodo = document.querySelector(".todo-inner");
+let isEdit = false;
+let editId = null;
+
+export default class display {
+  static getTodo = () => {
+    let Todo;
+    if (localStorage.getItem("todo") == null) {
+      Todo = [];
+    } else {
+      Todo = JSON.parse(localStorage.getItem("todo"));
+    }
+    return Todo;
+  };
+
+  static addRemoveEvent = () => {
+    const trash = document.querySelectorAll(".trash");
+    trash.forEach((task, i) => {
+      task.addEventListener("click", (ev) => {
+        ev.preventDefault();
+        this.removeTodo(i);
+      });
+    });
+  };
+
+  static removeTodo = (id) => {
+    const toDos = display.getTodo();
+    toDos.splice(id, 1);
```

Comment:
- [ ] [Optional] `Splice` method mutate the original array, which is a subject to a critical performance of your app in the worst case scenario, in order to avoid that issue, you can achieve the same by using functions or any other way which do not mutate the original array for example you can use `filter` function which returns a new array, it does not mutate the original array

Reply:
Oooh, @descholar-ceo  thanks for reviewing my work. I just wanted to inform you that this activity is part of the requirements for the next exercise. so I am requesting to do it in the next then if you are ok with it.